cluster/saltbase/salt/kube-dns/kubedns-rc.yaml.in

```diff
@@ -54,13 +54,21 @@ spec:
         args:
         # command = "/kube-dns"
         - --domain={{ pillar['dns_domain'] }}.
+        - --dns-port=10053
         ports:
         - containerPort: 53
           name: dns
           protocol: UDP
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+      - name: dnsmasq
+        image: gcr.io/google_containers/dnsmasq:1.0
+        command:
+        - dnsmasq
+        - -d
+        - -R
```

Comment:
can you use long-flag names here?  --no-resolv and --no-daemon. Docs say --no-daemon is for debug, and to use --keep-in-foreground instead. 

Do we also need --no-hosts ? 

Do we need --cache-size=  here?  Even if it is just informational, we might eventually set it from a configmap